agent/src/beerocks/monitor/monitor_thread.cpp

```diff
@@ -273,10 +273,10 @@ void monitor_thread::after_select(bool timeout)
 
             LOG(TRACE) << "mon_rssi.start()";
             if (!mon_rssi.start(&mon_db, slave_socket)) {
-                LOG(ERROR) << "mon_rssi.start() failed";
-                thread_last_error_code = MONITOR_THREAD_ERROR_ATTACH_FAIL;
-                stop_monitor_thread();
-                return;
+                // If monitor rssi failed to start, continue without it. It might failed due to
+                // insufficient permissions. Detailed error message is printed inside.
+                LOG(WARNING) << "mon_rssi.start() failed, ignore and continue without it";
```

Comment:
We should log the failure reason - i.e. if its because of missing permissions or some other failure. This has to be perfectly clear from this log.

Reply:
We don't need it here. A detailed error will be printed inside (as I wrote on the comment here).

Reply:
OK